posthog/hogql_queries/web_analytics/revenue_example_events.py

```diff
@@ -18,13 +24,21 @@ class RevenueExampleEventsQueryRunner(QueryRunner):
     response: RevenueExampleEventsQueryResponse
     cached_response: CachedRevenueExampleEventsQueryResponse
     paginator: HogQLHasMorePaginator
+    do_currency_conversion: bool = False
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.paginator = HogQLHasMorePaginator.from_limit_context(
             limit_context=LimitContext.QUERY, limit=self.query.limit if self.query.limit else None
         )
 
+        self.do_currency_conversion = posthoganalytics.feature_enabled(
```

Comment:
might be easier to just include this with the request and do it as a front end feature flag

Reply:
I think this will go away soon, so I'd rather keep it here for now. Some places need to do currency conversion, some don't, so this might be easier :)